internal/backuprestore/common.go

```diff
@@ -52,28 +54,17 @@ const (
 	clusterIDLabel   = "config.openshift.io/clusterID" // label for backups applied by lifecycle agent
 	defaultApplyWave = math.MaxInt32                   // 2147483647, an enough large number
 
-	defaultStorageSecret = "cloud-credentials"
-
-	oadpRestorePath = "OADP/veleroRestore"
-	oadpDpaPath     = "OADP/dpa"
-	oadpSecretPath  = "OADP/secret"
+	OadpRestorePath = "/opt/OADP/veleroRestore"
+	oadpDpaPath     = "/opt/OADP/dpa"
+	oadpSecretPath  = "/opt/OADP/secret"
 
 	// OadpNs is the namespace used for everything related OADP e.g configsMaps, DataProtectionApplicationm, Restore, etc
 	OadpNs = "openshift-adp"
 )
 
-// RestorePhase defines the phase of restore
-type RestorePhase string
-
-// Constants for restore phase
-const (
-	RestorePending    RestorePhase = "RestorePending"
-	RestoreFailed     RestorePhase = "RestoreFailed"
-	RestoreCompleted  RestorePhase = "RestoreCompleted"
-	RestoreInProgress RestorePhase = "RestoreInProgress"
-)
-
 var (
+	hostDir = "/host"
```

Comment:
nit: controllers/utils/constants.go has a Host constant already that can be used, although I see you're using a var so you can override it in tests. You could change this to initialize using the constant, then. But maybe we need to make that const a var in utils?

Reply:
Exactly, to use the one in controllers/utils/constants.go, I need to change it to var.  I didn't do that because I was afraid that the common variable be accidentally overridden.